CHANGELOG.md

```diff
@@ -25,11 +25,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * `Player::bus_name_player_name_part` - [Koen Bolhuis
   (InputUsername)](https://github.com/InputUsername)
 * `Metadata::as_hashmap(&self)` which returns a simple borrowed hashmap.
+* More `Player::has_*`, `Player::can_*`, and
+  `Player::checked_*` methods - [Harrison Thorne
+  (harrisonthorne)][harrisonthorne]
+  * `Player::has_volume`, `Player::checked_get_volume`,
+    `Player::checked_set_volume` 
+  * `Player::has_position`,
+    `Player::checked_get_position`,
+    `Player::checked_set_position`
+  * `Player::has_playback_rate`,
+    `Player::checked_get_playback_rate`,
+    `Player::checked_set_playback_rate` 
+  * `Player::can_loop`, `Player::checked_get_loop_status`
 
 ## Changed
 
 * `Player::checked_set_shuffle` also checks `::can_shuffle`. - [Stephan
   Henrichs (Kilobyte22)][Kilobyte22]
+* `Player::checked_set_loop_status` also checks `::can_loop` - 
+  [Harrison Thorne (harrisonthorne)][harrisonthorne]
+* `Progress` default values to used `checked_get_*`
+  functions - [Harrison Thorne
```

Comment:
Grammar could be improved here. I'll fix that myself when merging this, so don't worry about it!